src/content/docs/style-guide/article-templates/data-dictionary-style-guidelines.mdx

```diff
@@ -64,22 +40,46 @@ To add a new data type:
 
 To add attributes: 
 
-1. Check if there's an existing attribute that has the same name and same definition as the one you're trying to add. If it exists, edit that file to include the new data type. 
-2. If there is no existing attribute, copy an existing attribute file (for example, [this one](https://github.com/newrelic/docs-website/blob/develop/src/data-dictionary/events/Transaction/accountId.md)), edit all relevant fields to be accurate, and place it in the folder of a data type it's attached to (for example, [Transaction](https://github.com/newrelic/docs-website/tree/develop/src/data-dictionary/events/Transaction)). 
+1. Our goal is to avoid adding redundant attributes entries, so you should first check to see if there's an existing attribute that has the same name and same general definition as the one you're trying to add. If it exists, edit that file to include the new data type (under `events:`). If an existing attribute is fairly close in definition but not exact, try to edit the definition so it works for all associated data types.   
+2. If there is no existing attribute, create a new one. The easiest way is to copy an existing attribute file, edit all relevant fields to be accurate, and place it in the folder of the data type it's attached to (for example, [Transaction](https://github.com/newrelic/docs-website/tree/develop/src/data-dictionary/events/Transaction)). 
+3. When writing an attribute, try to keep it as concise as possible, and avoid using links. For more on that, see [Style](#style).  
+4. If the attribute has one of the units of measurement we use (like `units: percentage (%)`), add that to the front matter. For info on units, see [Units](#units).
+
+## Attribute style guidelines [#style]
+
+When adding or editing attributes, some things to keep in mind: 
 
-## How to set unit of measurement 
+* **Aim for concise descriptions.** Notes on this: 
```

Comment:
This section seems pretty wordy. Maybe we can trim some stuff or break it out into smaller, bulleted chunks?

Otherwise, I think your changes look good.

Reply:
I agree it's wordy. I just wanted to get some stuff out there quickly to prevent problems. Maybe we can finesse later.